tests/appsec/test_automated_login_events.py

```diff
@@ -289,8 +289,8 @@ def password_key(self):
         "Accept-Language": "en-GB, *;q=0.5",
         "Content-Language": "en-GB",
         "Content-Length": "0",
-        "Content-Type": "text/html; charset=utf-8",
-        "Content-Encoding": "deflate, gzip",
+        "Content-Type": "application/x-www-form-urlencoded; charset=utf-8",
```

Comment:
Changed to be the same that we are sending in the tests when we do a post request.